components/dashboard/src/components/PrebuildLogs.tsx

```diff
@@ -37,7 +37,7 @@ export default function PrebuildLogs(props: PrebuildLogsProps) {
         | undefined
     >();
     const [error, setError] = useState<Error | undefined>();
-    const [logsEmitter] = useState(new EventEmitter());
+    const logsEmitter = useMemo(() => new EventEmitter(), []);
```

Comment:
Just for confirmation: What's the effect you are after with this change?

Reply:
I was here when [initially trying](https://github.com/gitpod-io/gitpod/pull/20065/commits/fe6d61f51bbb8daafba27f8e1325ce257c8f6aaa) to make the buffering happen on the `ReplayableEventEmitter` side. I am happy to revert this, but I'd argue it improves readability of the component (when we need to touch it again) and better communicates intent → there is now a dependency array signalling that we don't ever want to change `logsEmitter`. Using `useState` felt like we're abusing it a bit

Reply:
Good, just wanted to make sure we are on the same page: Should make no difference :handshake: